django/contrib/admin/static/admin/js/app_filter.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+{
+    function initAppFilter() {
+        if (!document.getElementById('app-filter')) {
+            return;
+        }
+
+        const options = [];
+        const filterableElements = document.getElementsByClassName('filterable-apps-table');
+        for (const filterableElement of filterableElements) {
+            filterableElement.querySelectorAll('th[scope=row] a').forEach((container) => {
+                options.push({title: container.innerHTML, node: container});
+            });
+        }
+
+        function checkValue(event) {
+            let filterValue = event.target.value;
+            if (filterValue) {
+                filterValue = filterValue.toLowerCase();
+            }
+            if (event.key === 'Escape') {
+                filterValue = '';
+                event.target.value = ''; // clear input
+            }
+            let matches = false;
+            for (const o of options) {
+                let displayValue = '';
+                if (filterValue) {
+                    if (o.title.toLowerCase().indexOf(filterValue) === -1) {
+                        displayValue = 'none';
+                    } else {
+                        matches = true;
+                    }
+                }
+                // show/hide parent <TR>
+                o.node.parentNode.parentNode.style.display = displayValue;
+            }
+            if (!filterValue || matches) {
+                event.target.classList.remove('no-results');
+            } else {
+                event.target.classList.add('no-results');
+            }
+            sessionStorage.setItem('django.admin.navSidebarFilterValue', filterValue);
+        }
+
+        const nav = document.getElementById('app-filter');
+        nav.addEventListener('change', checkValue, false);
+        nav.addEventListener('input', checkValue, false);
+        nav.addEventListener('keyup', checkValue, false);
+
+        // Only retrieve the stored value if the current page is not the index page
+        if (!location.pathname.endsWith('/admin/')) {
```

Comment:
This will break if you don't host the admin on `/admin/` (e.g. `/obfuscated/`) right?